Bottom: 86b93b6ee0c24f69314bfc6dfd87e6579c27d253
Top:    6bfc45d9490ec1342328423a096c63715c29e945
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 10:41:13 -0400

Refresh of dspeak-audio

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index d480a73..2536f7c 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -44,14 +44,6 @@ config SND_SOC_MPC5200_I2S
 	help
 	  Say Y here to support the MPC5200 PSCs in I2S mode.
 
-config SND_MPC52xx_SOC_EFIKA
-	tristate "SoC AC97 Audio support for bbplan Efika and STAC9722"
-	depends on SND_MPC52xx_SOC && PPC_EFIKA
-	select SND_MPC52xx_SOC_AC97
-	select SND_SOC_STAC9766
-	help
-	  Say Y if you want to add support for sound on the Efika.
-
 config SND_MPC52xx_SOC_DSPEAK01
 	tristate "SoC I2S Audio support for Digispeaker DSPEAK01 with TAS5504"
 	depends on PPC_DSPEAK01
